src/components/dashboard/components/dictionary/DictionaryContainer.jsx

```diff
@@ -2,32 +2,58 @@ import React, { Component } from 'react';
 import propTypes from 'prop-types';
```

Comment:
Since the name of the class in this file is DictionaryOverview, the name of this file should also be DictionaryOverview

Reply:
This can be fixed in another pull request. Because am doing more than what my pull request is meant to do. What do you think?

Reply:
You might as well just raise another PR to fix it now.